src/Storage/Altinn.Platform.Storage.csproj

```diff
@@ -9,7 +9,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Altinn.Platform.Storage.Interface" Version="3.11.0" />
+    <PackageReference Include="Altinn.Common.AccessToken" Version="1.2.0" />
+	<PackageReference Include="Altinn.Common.AccessTokenClient" Version="1.1.3" />
```

Comment:
Is Storage in need of the client library?

Reply:
Nope, shouldn't be.

Reply:
Removed.